Samples/WinMLSamplesGallery/WinMLSamplesGallery/Samples/ImageClassifier/ImageClassifier.xaml

```diff
@@ -55,15 +55,12 @@
         </DataTemplate>
     </Page.Resources>
 
-    <Grid>
+    <Grid Margin="0,0,0,0">
         <Grid.RowDefinitions>
             <RowDefinition Height="Auto" />
             <RowDefinition Height="*" />
         </Grid.RowDefinitions>
-        <local_controls:SampleHeader

```

Comment:
>local_controls:SampleHeader [](http://example.com/codeflow?start=9&length=27)

Why remove this common control? Can we not include the title and description here?

Reply:
The SampleBasePage is currently holding the common template for the title of every sample so each sample file doesn't need to define their header.